app/models/attendance.rb

```diff
@@ -46,6 +48,10 @@ def calc_total_time_in_care
                               end
   end
 
+  def prevent_creation_of_absence_without_schedule
```

Comment:
@JonErvin is this something we still want to do?  Maybe we could talk through this later this week.

Reply:
Great question @katelovescode this was an open one on my end as well given the schedules convo.  Without a schedule, the info we need beyond date is # of hours the provider would expect the child to attend.  I think best case, we would prompt the user to share what they would bill (full day or # of hours 0-6).  In the interim, I think maybe we can use the same assumption we use on attendance (if no schedule, assume full day).  This might be a good team convo?

Reply:
@csprayregen since we're doing this iteratively I'm implementing this as initially planned - if there's no schedule they can't log an absence.  We can plan changes later!

Reply:
@katelovescode after yesterday's conversation, this should change, right? I'll updated the Wonderschool ticket to make sure this is covered too, if that makes sense.